src/Fhir.Anonymizer.Core/Extensions/ElementNodeVisitorExtensions.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Data;
+using System.Linq;
+using System.Text;
+using Fhir.Anonymizer.Core.Visitors;
+using Hl7.Fhir.ElementModel;
+
+namespace Fhir.Anonymizer.Core.Extensions
+{
+    public static class ElementNodeVisitorExtensions
+    {
+        public static void Accept(this ElementNode node, AbstractElementNodeVisitor visitor)
+        {
+            bool shouldVisitChild = visitor.Visit(node);
+
+            if (shouldVisitChild)
```

Comment:
Currently, this variable is always true right?

Reply:
Yes, currently it is always true.